src/overlays/actors/ovl_Door_Ana/z_door_ana.h

```diff
@@ -5,11 +5,30 @@
 
```

Comment:
Some overall naming suggestion. I don't like "direct" and "adjacent", since it doesn't describe what the difference is: one uses a scene exit, and the other is for an internal list into the generic grottos. I would suggest:
`DOORANA_TYPE_ADJACENT` -> `DOORANA_TYPE_SCENE_EXIT`
`GET_DOORANA_DIRECT_ENTRANCE` -> `GET_DOORANA_GROTTO_TYPE`
`GET_DOORANA_ADJACENT_ENTRANCE` -> `GET_DOORANA_SCENE_EXIT`
`GET_DOORANA_ROTATION_ENTRANCE` -> `GET_DOORANA_GROTTO_TYPE_EX` (this does feel like an "ex" scenario lol)